### ISSUE TITLE
Backspace doesn't work if Caps if locked.

### ISSUE THREAD

**Issue by pizdjuk** [Nheko-Reborn/nheko#1318]
### Describe the bug

Locked capital breaks the backspace key.

### To Reproduce

1. Write some text in the input field
2. Lock the capital (Shift+CapsLock or CapsLock, depending of xkb settings)
3. hit Backspace

Expected: backspace deletes characters
Observed: cursor not moving, characters wont delete

### Version

0.11.1-c13a9d3

### Operating system

Linux

### Installation method

Flathub

### Did you use profiles?

- [ ] Profiles used?

**Comment by LorenDB** [comment]
I can't reproduce this.

**Comment by deepbluev7** [comment]
Can you provide your keyboard layout or modifications you did to it? And possibly describe your system in more detail?

**Comment by pizdjuk** [comment]
> Can you provide your keyboard layout or modifications you did to it? And possibly describe your system in more detail?

actually nothing extraordinary. I have a cyrillic keyboard with switch on CapsLock. So if I want to lock a caps I press
Shift+CapsLock. So after doing this Backspace does not work. I suppose some other keys too.

System: Linux Mint, i3 as window manager, Xorg

**Comment by deepbluev7** [comment]
No, I can't reproduce it, so I have currently no idea how to fix it.

**Comment by pizdjuk** [comment]
> No, I can't reproduce it, so I have currently no idea how to fix it.

I just did some tests. The problem for me occurs in a flatpak version (I didnt try to compile the native version). And the problem occurs with a XCB backend:

```
$ flatpak run --env=QT_QPA_PLATFORM=xcb io.github.NhekoReborn.Nheko
```

As opposite: running the app in a weston wayland client for Xorg does not trigger the bug

```
$ weston &
$ flatpak run --env=QT_QPA_PLATFORM=wayland io.github.NhekoReborn.Nheko
```

**Comment by pizdjuk** [comment]
Any news?
